libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineViewTest.kt

```diff
@@ -1171,4 +1190,11 @@ class MapboxRouteLineViewTest {
         assertEquals(Visibility.VISIBLE, result)
         unmockkObject(MapboxRouteLineUtils)
     }
+
+    private fun getEmptyFeature(): Feature {
+        return Feature.fromJson(
+            "{\"type\":\"Feature\",\"id\":\"${UUID.randomUUID()}\"," +
```

Comment:
avoid using random values in tests as it non-determine behavior 
```suggestion
            "{\"type\":\"Feature\",\"id\":\"1\"," +
```

Reply:
That's exactly the reason a random number is being used here.